elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -68,14 +68,26 @@ trait RequestSigner {
   def withAuthHeader(httpRequest: HttpRequest): HttpRequest
 }
 
-trait RestlasticSearchClient extends ScrollClient {
+abstract class RestlasticSearchClient(searchExecutionCtx: ExecutionContext) extends ScrollClient {
```

Comment:
braking changes?

Reply:
Theoretically yes but I don't think it matters for anyone.